.github/workflows/tests.yml

```diff
@@ -0,0 +1,48 @@
+name: run-tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    name: test with ${{ matrix.py }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        py:
+          - "3.8"
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        include:
+          - py: "3.5"
```

Comment:
Let's go with 3.x (if vimdoc works on latest stable).

I cannot imagine us updating this (ever) + the current stable is 3.12.

If however it doesn't work on 3.12, let's open an issue and fix that.

Reply:
> If however it doesn't work on 3.12

Currently the tests don't, but that's just trivially because the nosetests runner seems to be in disrepair, nothing with the code or tests themselves. I'm planning to fix that soon after.